crap-code.kt

```diff
@@ -0,0 +1,26 @@
+package badcode.example
+class PerformanceFile {
+    val largeList = List(1000000) { it }
+    // Inefficient way to calculate the sum of elements
+    var sum = 0
+    fun calculateSum() {
```

Comment:
Avoid blocking the main thread with `Thread.sleep`; consider using asynchronous processing.
```suggestion
import kotlinx.coroutines.*

fun calculateSum() {
    GlobalScope.launch {
        for (i in 0 until largeList.size) {
            sum += largeList[i]
        }
        prentln("Sum: $sum")
        for (i in 0 until 100000) {
            val tempObject = Any()
        }
        var longString = ""
        for (i in 0 until 10000) {
            longString += "a"
        }
        println("Long String Length: ${longString.length}")
        delay(5000)
        println("Operation complete")
    }
}
```